src/www/views/servers_view/server_list/index.ts

```diff
@@ -55,7 +55,7 @@ export class ServerList extends PolymerElement {
       <template is="dom-repeat" items="[[servers]]">
         <server-card
           disabled="[[item.errorMessageId]]"
-          error-message="[[localize(item.errorMessageId)]]"
+          error-message="[[getErrorMessage(item.errorMessageId)]]"
```

Comment:
With Lit you could just inline the logic in the template.

Reply:
Yep - that's next!